src/pages/Homepage/Services/Services.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
```

Comment:
Kindly re-construct your file structuring, As you can see we do have views folder so you can create your UI structure here. Also as you can see under signup folder inside views folder i have to files 

signup_fields.js
signup.jsx

Kindly retain this way of coding starting this sprint.